heroku.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# Warning: this script has only been tested on macOS Sierra. There's a good chance
+# it won't work on other operating systems. If you get it working on another OS,
+# please send a pull request with any changes required. Thanks!
+set -e
+
+cd `dirname $0`
+r=`pwd`
+echo $r
+
+if [ -z "$(which heroku)" ]; then
+  echo "You must install the Heroku CLI first!"
+  echo "https://devcenter.heroku.com/articles/heroku-cli"
+  exit 1
+fi
+
+if ! echo "$(heroku plugins)" | grep -q heroku-cli-deploy; then
+  heroku plugins:install heroku-cli-deploy
+fi
+
+if ! echo "$(git remote -v)" | grep -q pwa-server-; then
+  server_app=pwa-server-$RANDOM
+  heroku create -r server $server_app
+else
+  server_app=$(heroku apps:info -r server --json | python -c 'import json,sys;obj=json.load(sys.stdin);print obj["app"]["name"]')
+fi
+serverUri="https://$server_app.herokuapp.com"
+
+if ! echo "$(git remote -v)" | grep -q pwa-client-; then
+  client_app=pwa-client-$RANDOM
+  heroku create -r client $client_app
+else
+  client_app=$(heroku apps:info -r client --json | python -c 'import json,sys;obj=json.load(sys.stdin);print obj["app"]["name"]')
+fi
+clientUri="https://$client_app.herokuapp.com"
+
+# Deploy the server
+cd $r/server
+mvn clean package -DskipTests
+
+# replace the client URL in the server
+sed -i -e "s|http://localhost:4200|$clientUri|g" $r/server/src/main/resources/application.properties
```

Comment:
Instead of doing this, I think we could add `--stormpath.web.cors.allowed.originUris=$clientUrl` to the `-o` options below. But idk, if it work this way it works.